.github/labels.yml

```diff
@@ -229,25 +218,13 @@
 - name: hint/needs-community-input
   color: 0623cc
   description: "Hint: Needs Community Input"
-- name: hint/needs-review
-  color: 0623cc
-  description: "Hint: Needs Review"
-- name: hint/needs-help
-  color: 0623cc
-  description: "Hint: Needs Help"  
 - name: hint/needs-docs
   color: 0623cc
   description: "Hint: Needs Docs"
 
 ###
 ### Statuses
 ###
-- name: status/done
-  color: edb3a6
-  description: "Status: Done"
-- name: status/approved-waiting
-  color: edb3a6
-  description: "Status: Approved and Awaiting Merge"
 - name: status/changes-requested
   color: edb3a6
   description: "Status: Changes Requested in Review"
```

Comment:
If we're getting rid of approved, etc., do we want to remove this one too?

Reply:
see pass 2!